src/sif/dynamiccost.cc

```diff
@@ -105,6 +105,7 @@ constexpr ranged_default_t<float> kClosureFactorRange{1.0f, kDefaultClosureFacto
 
 constexpr ranged_default_t<uint32_t> kVehicleSpeedRange{10, baldr::kMaxAssumedSpeed,
                                                         baldr::kMaxSpeedKph};
+constexpr ranged_default_t<uint32_t> kFixedSpeedRange{0, 0, baldr::kMaxSpeedKph};                                                    
```

Comment:
ah you already put it here.. 

hm, not so sure if 0 is a good default, I'll let others comment on that

Reply:
Thanks for all the feedback! I really appreciate it. Just wanted to explain the reasoning for 0 as a default. My thinking was I needed a value that was outside the range of viable numbers, so that there wouldn't be any limitations on what fixed speed you could set.  Zero made sense because if the fixed speed  is zero then you don't need any fixed speed routing you're at a standstill, you should use the estimated speed instead. Totally open to other options that was just my base reasoning.

Reply:
i think this will work fine however i would suggest making `0` a named constant so the check in the actual costing will be more readible

Reply:
one thing that is a bit strange from the API perspective is that you can actually set it to 0. i suppose the docs sh ould say valid values are from 1 to 160kph or whatever so people dont have to worry about what meaning 0 would have

Reply:
Sounds good made it a named constant called kDefaultFixedSpeed